backend/app/Models/Lesson.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Lesson extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'category_id', 'character'
+    ];
```

Comment:
This will work normally. What if we add new columns to the migration, we need to update also this one. Much better if we use `$guarded`
Ref: https://laravel.com/docs/9.x/eloquent#mass-assignment
topic: Laravel Mass assignment, guarded vs fillable 
```suggestion
    protected $guarded = [];
```